exe/genv-devices

```diff
@@ -240,8 +316,18 @@ def do_detach(devices: Devices, eid: str, index: Optional[int]) -> None:
 
 
 def do_ps(devices: Devices, format: str, header: bool, timestamp: bool) -> None:
+    """
+    Print genv devices data in a human-readable fashion to the stdio.
+    :param devices: dictionary describing all existing genv devices.
+    :param format: Which format should be used to print the devices data - csv or tui (default: tui).
```

Comment:
from the perspective of this very method, `format` does not have a default value

Reply:
Fixed, but adding a default value might be a good idea